src/__tests__/type-syncer.test.ts

```diff
@@ -161,20 +168,11 @@ function buildSyncer() {
           workspaces = rootPackageFile.workspaces
           break
         }
-        case 'packages/package-1/': {
-          workspaces = package1File.workspaces
-          break
-        }
-        case 'packages/package-2/': {
-          workspaces = package2File.workspaces
-          break
-        }
         default:
           throw new Error('What?!')
       }
 
-      workspaces ??= []
-      const globPromises = workspaces.map((w) =>
+      const globPromises = workspaces!.map((w) =>
```

Comment:
Yes, non-null assertations are bad, but I figured they fail fast, unlike `??=`.
Same thing with the mocks.